web/gds-user-ui/src/modules/dashboard/member/memberNetworkSelect.tsx

```diff
@@ -0,0 +1,18 @@
+import { FormControl, FormLabel, Select } from "@chakra-ui/react";
+import { Trans } from "@lingui/macro";
+
+const MemberSelectNetwork = () => {
+    return (
+    <FormControl>
```

Comment:
Using `FormControl` places the label above the select element. This is different from the Figma file, but I noticed that select components in the registration form followed this same style. If a change should be made, let me know.